### PR TITLE
Syndicate base converted to scene

### PR DESCRIPTION
**Pull request by ThatDan123**
Changes the syndicate base and shuttle into a scene.

Will only spawn scene if the gamemode has nuclear operatives as antags.

~Spawns the scene if admins manually spawn syndicates on non ops gamemode.~ Stopped until new system which doesnt freeze crash clients is figured out.

Supports specific scenes for maps if they want special syndicate base.

Supports random syndicate base being picked from a list of scenes.

All maps will need their syndicate stuff removed.

### CODE REVIEW

**Comment by ThatDan123** [comment]
Have an issue that freezes clients when the scene is spawned during a round.

**Comment by corp-0** [comment]
could it be the physics 2d sync Dooble has talked about before?

**Comment by ThatDan123** [comment]
I've stopped scene spawning during round for now.

File: UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/ReactionManager.cs [hunk 1]
```diff
@@ -14,8 +14,7 @@
 public class ReactionManager : MonoBehaviour
 {
 	public float n = 20;
-
-	[SerializeField]
+	
 	private GameObject fireLight = null;
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/ReactionManager.cs:18]: this one is not setteable in inspector anymore?

Comment (ThatDan123) [reply UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/ReactionManager.cs:18]: Doesnt need to be, its taken from the atmos manager instead

Comment (corp-0) [reply UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/ReactionManager.cs:18]: cool, are you gonna make any more changes? If not, I'm merging.

Comment (ThatDan123) [reply UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/ReactionManager.cs:18]: Cus every matrix has its own reaction manager, while theres only one atmos manager. I just cache it there instead

Comment (ThatDan123) [reply UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/ReactionManager.cs:18]: Should be good to merge

**Pull request merged**